Workleap.DotNet.CodingStandards.nuspec

```diff
@@ -14,6 +14,8 @@
     <repository type="git" url="$RepositoryUrl$" commit="$RepositoryCommit$" branch="$RepositoryBranch$" />
     <dependencies>
       <dependency id="Microsoft.CodeAnalysis.BannedApiAnalyzers" version="3.3.4" />
+      <dependency id="Microsoft.CodeAnalysis.NetAnalyzers" version="8.0.0" />
```

Comment:
Including Microsoft.CodeAnalysis.NetAnalyzers as part of the dependencies so it can flow on consumer projects that are using an older version